docs/4.1/admin-guide.md

```diff
@@ -2435,28 +2397,23 @@ $ teleport start --diag-addr=127.0.0.1:3000
 Now you can see the monitoring information by visiting several endpoints:
 
 * `http://127.0.0.1:3000/metrics` is the list of internal metrics Teleport is
-
    tracking. It is compatible with [Prometheus](https://prometheus.io/)
    collectors.
 
 * `http://127.0.0.1:3000/healthz` returns "OK" if the process is healthy or
-
-`503` otherwise.
+  `503` otherwise.
 
 * `http://127.0.0.1:3000/readyz` is similar to `/healthz` , but it returns "OK"
-
   _only after_ the node successfully joined the cluster, i.e.it draws the
   difference between "healthy" and "ready".
 
 * `http://127.0.0.1:3000/debug/pprof/` is Golang's standard profiler. It's only
-
   available when `-d` flag is given in addition to `--diag-addr`
 
 ## Getting Help
 
-Please open an [issue on
-Github](https://github.com/gravitational/teleport/issues). Alternatively, you
-can reach through the contact form on our [website](https://gravitational.com/).
+Please open an [issue on Github](https://github.com/gravitational/teleport/issues). Alternatively,
+you can reach through the contact form on our [website](https://gravitational.com/).
```

Comment:
The new recommendation is to use the Community forums (https://community.gravitational.com) for help rather than opening a Github issue.